CMakeLists.txt

```diff
@@ -502,12 +502,15 @@ else()
       # -DCHOST=... or -DCMAKE_AR=... and -DCMAKE_RANLIB=...
```

Comment:
This comment needs fixing now: to "... specify -DCHOST and optionally -DCMAKE_AR and -DCMAKE_RANLIB if they need to be different from ${CHOST}-gcc{ar,ranlib}"